R/importMetaphlan.R

```diff
@@ -3,19 +3,20 @@
 #' @param file a single \code{character} value defining the file
 #'   path of the Metaphlan file. The file must be in merged Metaphlan format.
 #'
-#' @param colData a DataFrame-like object that includes sample names in
+#' @param col.data a DataFrame-like object that includes sample names in
 #'   rownames, or a single \code{character} value defining the file
 #'   path of the sample metadata file. The file must be in \code{tsv} format
-#'   (default: \code{colData = NULL}).
+#'   (default: \code{col.data = NULL}).
 #' 
-#' @param sample_meta a DataFrame-like object that includes sample names in
-#'   rownames, or a single \code{character} value defining the file
-#'   path of the sample metadata file. The file must be in \code{tsv} format
-#'   (default: \code{sample_meta = NULL}).
+#' @param colData Deprecated. use \code{col.data} instead.
+#'   
+#' @param sample_meta Deprecated. Use \code{col.data} instead.
 #' 
-#' @param phy_tree a single \code{character} value defining the file
+#' @param phy.tree.file a single \code{character} value defining the file
 #'   path of the phylogenetic tree.
-#'   (default: \code{phy_tree = NULL}).
+#'   (default: \code{phy.tree.file = NULL}).
+#' 
+#' @param phy_tree Deprecated. Use \code{phy.tree.file} instead.
```

Comment:
The name could be just tree.file

Reply:
Check also other importers. The name should be harmonized in all

Reply:
ACK.